resources/js/Pages/Auth/Signup.vue

```diff
@@ -0,0 +1,69 @@
+
+<script setup>
+import { useForm } from '@inertiajs/vue3'
+import { watch } from 'vue'
+const form = useForm({
+  name: '',
+  email: '',
+  password: '',
+  errors: {},
+})
+
+const submit = async () => {
+  try { 
+    form.post('/register') 
+  } catch (error) {
+    console.log(error)
+  }
+}
+watch(() => form.errors, (errors) => {
+  console.log(errors)
+})
```

Comment:
Do we need this `watch` and `console.log()` here and above?

Reply:
okay, i get rid of it